styles.css

```diff
@@ -1,880 +1,359 @@
-    width: 95%;
-    border-radius: 8px;
-  }
-
-  .images1:hover {
-    animation: left-to-bottom 3s;
-  }
-
-  @keyframes left-to-bottom {
-    100% {
-      transform: translate(50%);
-    }
-  }
-
-  .flexbox-item3-6 {
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    justify-content: flex-start;
-    align-items: flex-start;
-    color: #344563;
-  }
-
-  .contact-container {
-    width: 100%;
-    background: #6070ff;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    border-top-left-radius: 80px;
-  }
-
-  .contact-item-1 {
-    position: static;
-    width: 40%;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 40px;
-    line-height: 52px;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    color: #fff;
-    justify-content: center;
-  }
-
-  .contact-item-2 {
-    position: static;
-    width: 30%;
-    left: 116px;
-    top: 68px;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 20px;
-    line-height: 28px;
-    text-align: center;
-    color: #ebebff;
-  }
-
-  .form {
-    display: flex;
-    align-items: center;
-    background: #fff;
-    border: 1px solid #cfd8dc;
-    border-radius: 2px;
-    padding: 10px;
-    width: 100%;
-  }
-
-  .form-btn {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    background: #fff;
-    border-radius: 8px;
-    padding: 12px 16px;
-    width: 40%;
-    justify-content: center;
-  }
-
-  .form-btn:hover {
-    background-color: green;
-    color: #fff;
-  }
-
-  .form-btn:active {
-    background-color: yellow;
-    color: #fff;
-  }
-
-  .bullet {
-    font-family: "Poppins", sans-serif;
-    list-style-type: none;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 18px;
-    line-height: 24px;
-    color: #344563;
-  }
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Poppins", sans-serif;
+  background: #E5E5E5;
+}
+
+header {
+  background-color: white;
+  padding: 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items:flex-start;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+  margin-left: 10%;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+header ul {
+  list-style-type: none;
+  display: flex;
+  color: #344563;
+  font-weight: 500;
+  font-size: 15px;
+  padding-right: 10%;
+}
+
+header li {
+  padding-right: 10%;
+}
+
+.row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+[class*="col-"] {
+  float: left;
+  padding: 15px;
+}
+
+article {
+  background-image: url("/imgs/background-desktop.png");
+  border-bottom-left-radius: 80px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ] Kindly your deployment is working very well just one thing is the background here is not appear, Kindly fix it.

![image](https://user-images.githubusercontent.com/35535130/166302318-2ca3b75b-fa70-47d4-b5eb-8f8250098ea3.png)

Reply:
@keroloussamy , The suggested improvement has been incorporated. Thank you